src/lib/dev-environment/dev-environment-lando.ts

```diff
@@ -89,6 +89,10 @@ async function getLandoConfig(): Promise< LandoConfig > {
 		home: fakeHomeDir,
 		domain: 'vipdev.lndo.site',
 		version: 'unknown',
+		env: {
+			LANDO_HOST_USER_ID: process.platform === 'win32' ? '1000' : `${ userInfo().uid }`,
+			LANDO_HOST_GROUP_ID: process.platform === 'win32' ? '1000' : `${ userInfo().gid }`,
```

Comment:
These variables will be passed to `docker compose`; it will use them for substitutions inside `docker-compose` files.

See https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/ for details.

Reply:
Does this need Windows native and WSL2 testing?

Reply:
No, this was stolen from Lando:

https://github.com/Automattic/lando-cli/blob/6ca2668273b1c4d07bc293d0c49f1a56646c03ac/lib/user.js#L20